### DISCUSSION TITLE
Github Pages sub-sites

### DISCUSSION THREAD

**Discussion by Galaxy-Coding** [community/community#22670] [Other Feature Feedback, Questions, & Ideas]
<p>I have a repository at <a href="https://github.com/Galaxy-Coding/galaxy-coding.github.io" rel="nofollow noopener">https://github.com/Galaxy-Coding/galaxy-coding.github.io</a> and it is published using Github Pages to<a href="https://galaxy-coding.github.io/" rel="nofollow noopener">https://galaxy-coding.github.io/</a>. I would like to add<code>https://galaxy-coding.github.io/ SUBSITE</code> to my url without the .html or .htm extension. How would I do that?<br>
Thank you in advance</p>

**Comment by piramiday** [comment]
<p>instead of creating a file titled <code>SUBSITE.html</code> in your root folder, try creating a subfolder named <code>SUBSITE</code> and then put an <code>index.html</code> file inside it, e.g.: <code>SUBSITE/index.html</code></p>

**Comment by Galaxy-Coding** [comment]
<p>Great answer but I would like to know i I have several subsites in one folder like:<br>
*/subsite/1<br>
*/subsite/2<br>
*/subsite/3</p>

**Comment by piramiday** [comment, marked as answer]
<p>what are you trying to accomplish?</p>
<p>if you want to have a single folder, <code>SUBSITE</code>, and three files, <code>1.html</code>, <code>2.html</code>, <code>3.html</code>, then you can of course, but you will have to type the full URLs:</p>
<ul>
<li>username github io/SUBSITE/1.html</li>
<li>username github io/SUBSITE/2.html</li>
<li>username github io/SUBSITE/3.html</li>
</ul>
<p>if, instead, your goal is to have these three working URLs:</p>
<ul>
<li>username github io/SUBSITE/1</li>
<li>username github io/SUBSITE/2</li>
<li>username github io/SUBSITE/3</li>
</ul>
<p>then you can simply put your <code>index.html</code> files with yet another subfolder, each. e.g.:</p>
<pre><code class="lang-auto">/SUBSITE/1/index.html
/SUBSITE/2/index.html
/SUBSITE/3/index.html
</code></pre>

**Reply by 0yqc** [reply]
I know this is long over, but for anyone googling...

You can use permalink: /:name/ in _config.yml so github.io/a.html can be accecesed on github.io/a

Or has this any disadvantages?

**Reply by 0yqc** [reply]
Yes, a few month of coding later, this has significant disadvantages:
- A local server wouldn't work
- the url github.io/a/ will be 404 not found, only github.io/a will work.
-
